source/includes/limitations-rs-to-sharded.rst

```diff
@@ -0,0 +1,12 @@
+- ``mongosync`` allows users to rename collections that the
+  ``sharding.shardingEntries`` option includes during sync. To see limitations 
+  on renaming collections while ``mongosync`` is running, see :ref:`Renaming
+  During Sync <rename-during-sync>`.
+- When using the ``sharding.createSupportingIndexes`` option to create
```

Comment:
[suggestion] for more [user centric language](https://www.mongodb.com/docs/meta/style-guide/writing/write-to-the-user/#write-to-the-user)

```suggestion
- If you use the ``sharding.createSupportingIndexes`` option to create
```

Reply:
Thanks for the suggestion, changing now!